### PR TITLE
Snap 747 jenkins failures

### PR DESCRIPTION
**Pull request by money**
### Jira Story

- [Jenkins is failing all Intake PRs](https://osi-cwds.atlassian.net/browse/SNAP-747)

## Description
Debug the failing specs in Intake and implement fixes.

## Tests
- [ ] I have included unit tests 
- [x] I have included feature tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

## Types of changes
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the code style of this project.
- [x] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by money** [comment]
Jenkins retest this please

**Comment by money** [comment]
Jenkins retest this please

**Comment by money** [comment]
Special thanks to @kelfink for his assist in adding Jenkins config with Docker to see failing specs in html/png formats.

jenkins

File: spec/features/screening/participant/editing_participant_spec.rb [hunk 1]
```diff
@@ -567,7 +567,9 @@
         fill_in_react_select('Role', with: 'Non-mandated Reporter')
         expect(page).to have_react_select_field('Role', with: ['Mandated Reporter'])
 
-        remove_react_select_option('Role', 'Mandated Reporter')
```
Comment (wmitchell-cambria) [inline spec/features/screening/participant/editing_participant_spec.rb]: Are these helper methods still useful in some situations, or are they just broken?

File: spec/features/screening/validations/screening_decision_validations_spec.rb [hunk 2]
```diff
@@ -274,60 +274,66 @@
       let(:screening_decision) { nil }
 
       scenario 'displays no error on initial load' do
-        should_not_have_content error_message, inside: '#decision-card.edit'
```
Comment (wmitchell-cambria) [inline spec/features/screening/validations/screening_decision_validations_spec.rb]: Same question here.

File: spec/features/screening/participant/editing_participant_spec.rb (hunk 1 above)
Comment (money) [reply spec/features/screening/participant/editing_participant_spec.rb]: This seems broken

File: spec/features/screening/validations/screening_decision_validations_spec.rb (hunk 2 above)
Comment (money) [reply spec/features/screening/validations/screening_decision_validations_spec.rb]: Not broken. Just added step for some reason.

**Pull request merged**